src/app/project/manage-payments/manage-payments.component.ts

```diff
@@ -1,61 +1,67 @@
 })
 export class ManagePaymentsComponent implements OnInit {
 
-  projectWallet: WalletModel;
-  projectModel: ProjectModel;
+    @Input() revenueShareAmount;
+    projectWallet: WalletModel;
+    projectModel: ProjectModel;
 
-  constructor(private managePaymentsService: ManagePaymentsService,
-              private route: ActivatedRoute, 
-              private projectService: ProjectService) {
-  }
+    constructor(private managePaymentsService: ManagePaymentsService,
+                private route: ActivatedRoute,
+                private router: Router,
+                private projectService: ProjectService) {
+    }
 
-  ngOnInit() {
+    ngOnInit() {
+        const projID = this.route.snapshot.params.projectID;
+        this.getProjectWallet(projID);
+        this.getProject(projID);
+    }
 
-    const projID = this.route.snapshot.params.projectID;
+    getProject(projectID: string) {
+        SpinnerUtil.showSpinner();
+        this.projectService.getProject(projectID).subscribe((res: any) => {
+            SpinnerUtil.hideSpinner();
+            this.projectModel = res;
+        }, err => {
+            SpinnerUtil.hideSpinner();
+            displayBackendError(err);
+        });
+    }
 
-    this.getProjectWallet(projID);
-    this.getProject(projID);
-  }
+    getProjectWallet(projectID: number) {
+        SpinnerUtil.showSpinner();
+        this.managePaymentsService.getProjectWallet(projectID).subscribe((res: any) => {
+            SpinnerUtil.hideSpinner();
+            this.projectWallet = res;
+            this.projectWallet.currency = prettyCurrency(res.currency);
+            this.projectWallet.balance = numeral(res.balance).format('0,0');
+            autonumericCurrency('#revenueShareAmount');
+        }, err => {
+            SpinnerUtil.hideSpinner();
+            displayBackendError(err);
+        });
+    }
 
-  getProject(projectID: string) {
-    SpinnerUtil.showSpinner();
-    this.projectService.getProject(projectID).subscribe((res: any) => {
-      SpinnerUtil.hideSpinner();
-      this.projectModel = res;
-    }, err => {
-      SpinnerUtil.hideSpinner();
-      displayBackendError(err);
-    });
-  }
-
-  getProjectWallet(projectID: number) {
-
-
-    SpinnerUtil.showSpinner();
-    this.managePaymentsService.getProjectWallet(projectID).subscribe((res: any) => {
-      SpinnerUtil.hideSpinner();
-      this.projectWallet = res;
-      this.projectWallet.currency = prettyCurrency(res.currency);
-      this.projectWallet.balance = numeral(res.balance).format('0,0');
-    }, err => {
-      SpinnerUtil.hideSpinner();
-      displayBackendError(err);
-    });
-  }
+    startPayout() {
+        const projID = this.route.snapshot.params.projectID;
+        const orgID = this.route.snapshot.params.groupID;
 
+        this.router.navigate(['/dash', 'manage_groups', orgID.toString(),
+            'manage_project', projID, 'manage_payments', 'revenue_share', this.revenueShareAmount]);
```

Comment:
You can use string interpolation method because it's usually more readable.

```suggestion
        this.router.navigate(`/dash/manage_groups/${orgID}/manage_project/${projID}/manage_payments/revenue_share/${this.revenueShareAmount}`);
```

Please, check the line length, I'm not sure how if it fits. 😆